src/containers/ProfileDetail/ContentSection/ContentSection.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState, MouseEvent } from 'react';
+
+import StudyTab from '../StudyTab/StudyTab';
+import PostTab from '../PostTab/PostTab';
+
+import { Nav, Button } from './ContentSection.style';
+
+function ContentSection() {
+  const [tab, setTab] = useState<'study' | 'post'>('study');
+
+  const handleClick = (e: MouseEvent) => {
+    if (e.target instanceof HTMLButtonElement) {
+      const newTab = e.target.dataset.value as 'study' | 'post';
+      setTab(newTab);
+    }
+  };
```

Comment:
```suggestion
  const handleClick = (e: MouseEvent<HTMLButtonElement>) => {
    if (e.target) {
      const newTab = e.target.dataset.value as 'study' | 'post';
      setTab(newTab);
    }
  };
```
instancefof 보다 제네릭으로 element 지정해서 이벤트 받아올 수 있지 않나요...?